### PR TITLE
[ADD]Messages to user endpoints

### PR DESCRIPTION
**Pull request by ealexisaraujo**

### CODE REVIEW

File: components/users/routes.js [hunk 1]
```diff
@@ -79,7 +80,7 @@ function usersApi(app) {
 
   router.post(
     '/',
-    passport.authenticate('jwt', { session: false }),
+    // passport.authenticate('jwt', { session: false }),
```
Comment (hyfi06) [inline components/users/routes.js:83]: :eyes:

File: components/users/routes.js [hunk 2]
```diff
@@ -112,9 +113,27 @@ function usersApi(app) {
...
+            } else if (criteria === 'isActive' && user[criteria] == false) {
+              return `User disabled sucessfull`;
+            } else {
+              return `Data updated`;
+            }
+          });
+
+        if (message.length == 0) {
+          throw boom.badRequest('Please verify field you want to update');
+        }
```
Comment (hyfi06) [inline components/users/routes.js:132]: this validation should be before call `usersService` or in first line of method `updateUser`

File: components/users/routes.js [hunk 3]
```diff
@@ -112,9 +113,27 @@ function usersApi(app) {
           user,
         });
 
+        const query = Object.keys(user);
+
+        const message = query
```
Comment (hyfi06) [inline components/users/routes.js:118]: a suggest:
```js
const message = query.includes('isActive') ? 
      `User ${user.isActive == 'true' ? 'enabled': 'disabled' } sucessfull` :
       'Data updated';
```

File: components/users/routes.js [hunk 4]
```diff
@@ -112,9 +112,17 @@ function usersApi(app) {
...
+        const message = query.includes('isActive')
+          ? `User ${
+              user.isActive == 'true' ? 'enabled' : 'disabled'
+            } sucessfull`
+          : 'Data updated sucessfully';
+
         res.status(200).json({
           data: updatedUser,
-          message: 'user updated',
+          message: `${message}`,
```
Comment (hyfi06) [inline components/users/routes.js:125]: `message` is a String, the template literals not required

Comment (ealexisaraujo) [reply components/users/routes.js]: Done 🔥

**Pull request merged**